distributed/worker.py

```diff
@@ -3111,6 +3122,14 @@ def get_client(address=None, timeout=3, resolve_address=True):
     worker_client
     secede
     """
+
+    # Get timeout value from dash config if not passed.
+    # Defaults to 3s if the config is not set.
+    if timeout is None:
+        timeout = parse_timedelta(
+            dask.config.get("distributed.comm.timeouts.connect", "3s"), "s"
+        )
```

Comment:
We probably want to support `get_client(timeout="10s")` as well, so it might be best to wrap the `parse_timedelta` outside of the conditional.  The same is probably true elsewhere as well.

Reply:
Thanks for the suggestion, Made the requested changes.